packages/frontend/src/components/LightdashVisualization/VisualizationConfigPie.tsx

```diff
@@ -47,15 +47,7 @@ const VisualizationPieConfig: FC<VisualizationConfigPieProps> = ({
 }) => {
     const { dimensions, numericMetrics } = useMemo(() => {
         const metrics = getMetricsFromItemsMap(itemsMap ?? {}, isNumericItem);
-        const tableCalculations = Object.entries(itemsMap ?? {}).reduce<
-            Record<string, TableCalculation>
-        >((acc, [key, value]) => {
-            if (isTableCalculation(value)) {
-                return { ...acc, [key]: value };
-            }
-            return acc;
-        }, {});
-
+        const tableCalculations = getTableCalculationsFromItemsMap(itemsMap);
```

Comment:
I can split this out if anyone wants. We should clean up how everything is getting these